cmake_toolchain/vita.cmake

```diff
@@ -23,13 +23,33 @@ include(CMakeParseArguments)
 ## MACRO: vita_create_self
 ##
 ## Generate a SELF from an ARM EABI ELF
-##   vita_create_self(target source
+##   vita_create_self(self source
 ##                    [CONFIG file]
 ##                    [UNCOMPRESSED]
 ##                    [UNSAFE])
 ##
+macro(vita_create_self self source)
+  if(${CMAKE_GENERATOR} MATCHES ".*Ninja.*")
```

Comment:
is this value more good than `CMAKE_MAKE_PROGRAM`?

Reply:
Yes.
> The name of the generator that is being used to generate the build files. (e.g. Unix Makefiles, Ninja, etc.)
The value of this variable should never be modified by project code. A generator may be selected via the cmake(1) -G option, interactively in cmake-gui(1), or via the CMAKE_GENERATOR environment variable.

compare with

> Tool that can launch the native build system. The value may be the full path to an executable or just the tool name if it is expected to be in the PATH.